wafer/sponsors/views.py

```diff
@@ -20,3 +23,15 @@ class SponsorView(DetailView):
 class ShowPackages(ListView):
     template_name = 'wafer.sponsors/packages.html'
     model = SponsorshipPackage
+
+
+class SponsorViewSet(viewsets.ModelViewSet):
+    """API endpoint for users."""
+    queryset = Sponsor.objects.all()
+    serializer_class = SponsorSerializer
```

Comment:
Do we want to add `permission_classes = (DjangoModelPermissionsOrAnonReadOnly, )` here?